pylti/flask.py

```diff
@@ -106,7 +106,7 @@ def _verify_any(self):
         except LTINotInSessionException:
             self.verify_request()
 
-    def _verify_session(self):
+    def _verify_session(self):  # pylint: disable=no-self-use
```

Comment:
if this doesn't have self use, you should decorate it with `@staticmethod` and remove self from the arguments